sections/installation.js

```diff
@@ -145,23 +145,6 @@ export default [
     content: "Initializing the library should normally only ever happen once in your application, as it creates a connection pool for the current database, you should use the instance returned from the initialize call throughout your library."
   },
   {
-    type: "text",
-    content: "You can even use knex without a connection, just for its query building features. Just pass in an empty object when initializing the library. Specify a client if you are interested in a particular flavour of SQL."
-  },
-  {
-    type: "code",
-    language: "js",
-    content: `
-      var knex = require('knex')({});
-      var pg = require('knex')({client: 'pg'});
```

Comment:
This is still possible. Only passing empty object without client attribute is not possible anymore.